src/elements/JobInternalArtifacts/JobInternalArtifacts.js

```diff
@@ -0,0 +1,131 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Link } from 'react-router-dom'
+import prettyBytes from 'pretty-bytes'
+import { connect } from 'react-redux'
+
+import jobsActions from '../../actions/jobs'
+import { formatDatetime } from '../../utils'
+
+import downloadIcon from '../../images/download-icon.png'
+
+const JobInternalArtifacts = ({ jobsStore, getArtifacts }) => {
+  const items = jobsStore.selectedJob.artifacts.map(item => {
+    const index = item.target_path.indexOf('://')
+    const target_path = {
+      schema: item.target_path.includes('://')
+        ? item.target_path.slice(0, index)
+        : null,
+      path: item.target_path.includes('://')
+        ? item.target_path.slice(index + '://'.length)
+        : item.target_path
+    }
+    return {
+      key: item.key,
+      target_path: target_path,
+      size: prettyBytes(item.size),
```

Comment:
We need to handle the case where `size` is missing. In this case we would like to display “size: N/A” in the table cell.